index.html

```diff
@@ -81,8 +81,9 @@ <h2>Protect your data and code in use and at rest</h2>
     <div class="row no-gutters">
         <div class="col-12 col-md-6">
             <h2>Want to learn more?</h2>
-            <p>Get started with Asylo by running through our quickstart guide.</p>
-            <a class="btn btn-asylo" href="{{home}}/docs/guides/quickstart.html">GET STARTED</a>
+            <p>Get started with Asylo via our quickstart guide, or our free Qwiklabs exercises.</p>
```

Comment:
As far as I'm aware, Qwiklabs are not free (must pay for the associated GCP resources, to avoid abuse - e.g., cryptocurrency mining). There is code currently provided that removes the need to pay for those resources for our labs, but it expires 2019-04-01 (when C3 ends).

Reply:
Removed "free"